### ISSUE TITLE
Lutein and Antioxidants Measure showing (nearly) zero

### ISSUE THREAD

**Issue by brianmackenna** [ebmdatalab/openprescribing#2617]
The [lutein and antioxidant measure](https://openprescribing.net/measure/lplutein/national/england/#lplutein) is showing (nearly) zero  since October. 

I have checked BQ and it appears raw data has these low numbers
<details><summary>Code snippet</summary>
<p>

```SELECT 
     SUM(actual_cost) AS cost,
     SUM(quantity) AS QTY,
     SUM(items) AS item
 FROM ebmdatalab.hscic.prescribing_2020_01
 WHERE bnf_code IN ("091000000BBTAA0", "091000000BBTPA0", "091000000BBUQA0", "091000000BBUSA0", "091000000BBUZA0", "091000000BBVFA0", "091000000BBVGA0", "091000000BBVQA0", "091000000BBVSA0", "091000000BBVTA0", "091000000BBVXA0", "091000000BBVZA0", "091000000BBWHA0", "091000000BBWNA0", "091000000BBWPA0", "091000000BBWQA0", "091000000BBWVA0", "091000000BBWWA0", "091200000BEJYA0", "091200000BEXXA0", "091200000BFLHA0")
GROUP BY
practice
```

</p>
</details>

This makes me think it might be to do with change in BNF mapping @inglesp ? And we are still awaiting the new mapping file from BSA.

Unless someone else knows the reason off top of their head this is low priority for investigation until at least after BSA release the new file.

**Comment by inglesp** [comment]
Nothing to do with the BNF/SNOMED mapping, since `lplutein` is defined in terms of BNF codes and not SNOMED codes.

Looks like most of the presentations in the measure definition have changed their BNF code between 2019_09 and 2019_10.

For instance, "PreserVision tablets" used to have BNF code `091000000BBTPA0` and now have BNF code `091000000BBTPBU`.

In the past, we have been given a file that lists all such BNF code changes by the BSA.  I'll ask whether they can provide us with the latest list of BNF code changes.

**Comment by brianmackenna** [comment]
The BSA has provided new BNF code to old BNF code maps at https://www.nhsbsa.nhs.uk/bnf-code-changes

This has been changed in the database but we still need to rewrite the measure definition for [lutein and antioxidants](https://github.com/ebmdatalab/openprescribing/blob/master/openprescribing/measure_definitions/lplutein.json).

**Comment by richiecroker** [comment]
Closed via #3256
